azure-pipelines.yml

```diff
@@ -234,9 +234,9 @@ jobs:
       displayName: Restore
       inputs:
         filePath: eng/build.ps1
-        arguments: -configuration Debug -prepareMachine -ci -restore -binaryLog
+        arguments: -configuration Debug -prepareMachine -ci -useGlobalNuGetCache:$false -restore -binaryLog
 
-    - powershell: .\eng\test-rebuild.ps1 -ci -configuration Release
+    - powershell: .\eng\test-rebuild.ps1 -ci -useGlobalNuGetCache:$false -configuration Release
```

Comment:
Do you still need this now that you've removed the problematic reference?

Reply:
Good catch. I'll remove the arguments but I'll leave the parameter in the test-rebuild script alone for now.